### PR TITLE
Remove mentions of the tablets

### LINKED ISSUE #63: Remove references to the tablets
We already have docs on how to use other devices to connect to the WiFi, so we just need to remove the mentions of the tablets themselves to avoid confusion.

This follows the [decision](https://github.com/srobo/core-team-minutes/blob/049aca468627efdc0d204218f94f5a645384930a/2018/2018-10-30.md#minutes) not to ship the tablets for SR2019.

### PR DESCRIPTION
**Pull request by PeterJCLaw**
Fixes #63.

I suggest reviewing by commit as the early commits do some related tidyup.

### CODE REVIEW

File: kit/wifi.md [hunk 1]
```diff
@@ -46,7 +44,7 @@ The stop button that appears after starting your code is currently not functiona
 
 ### Setting up the Robot's Environment
 
-![The robot interface's set up screen]({{ site.baseurl }}/images/content/kit/tablet/set-up-screen.png)
+![The robot interface's set up screen]({{ site.baseurl }}/images/content/kit/remote-interfaceg/set-up-screen.png)
```
Comment (RealOrangeOne) [inline kit/wifi.md]: Looks like there's a stray `g` in this path

File: kit/wifi.md [hunk 2]
```diff
@@ -62,7 +60,7 @@ You cannot use this screen when code is running.
 
 ### Logs Screen
 
-![The robot interface's logs screen]({{ site.baseurl }}/images/content/kit/tablet/logs-screen.png)
+![The robot interface's logs screen]({{ site.baseurl }}/images/content/kit/remote-interfaceg/logs-screen.png)
```
Comment (RealOrangeOne) [inline kit/wifi.md]: And this path

File: kit/wifi.md (hunk 1 above)
Comment (PeterJCLaw) [reply kit/wifi.md]: 0617320

File: kit/wifi.md (hunk 2 above)
Comment (PeterJCLaw) [reply kit/wifi.md]: 0617320

**Pull request merged**